topics/userscript/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: userjs, greasemonkey, tampermonkey, violentmonkey
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: March 28, 2005
+short_description: A kind of lightweight browser extension, used to modify webpages.
```

Comment:
Could you rephrase this as a full sentence? Maybe "A userscript is a kind of lightweight browser extension that is used to modify webpages."

Reply:
done